Add warnings for Cloud tasks requiring redeploy and misc edits (#6766)
magento/devdocs: src/cloud/live/stage-prod-migrate.md

   mysqldump -h <database host> --user=<database username> --password=<password> --single-transaction --triggers <database name> | gzip - > /tmp/database.sql.gz
   ```

1. Transfer the database dump to another remote environment with the `rsync` command:

   ```bash
   rsync -azvP /tmp/database.sql.gz <destination_environment_ssh_link@ssh.region.magento.cloud>:/tmp
   ```

> 1. Type `exit` to terminate the SSH connection.

Author: ```suggestion
1. Type `logout` to close the SSH connection.
```

> 1. Type `exit` to terminate the SSH connection.

Author: ```suggestion
1. Type `logout` to terminate the SSH connection.
```

> 1. Type `exit` to terminate the SSH connection.

Author: ```suggestion
1. Type `logout` to terminate the SSH connection.
```
